app/controllers/images_controller.rb

```diff
@@ -244,4 +254,26 @@ def update_crop_params
     # FIXME: this will raise a warning because of unpermitted paramaters
     params.permit(:crop_x, :crop_y, :crop_width).merge(crop_height: crop_height.to_i)
   end
+
+  def next_lead_image_revision(revision, image_revision, selected)
+    if revision.lead_image_revision&.image_id == image_revision.image_id
+      return nil unless selected
```

Comment:
This would be easier to read if the selected was part of the if statement - it's a little confusing to enter the block and then do nothing

Also, minor but the `nil` is redundant here